service/history/fx.go

```diff
@@ -81,6 +82,7 @@ var Module = fx.Options(
 	fx.Provide(HistoryEngineFactoryProvider),
 	fx.Provide(HandlerProvider),
 	fx.Provide(ServiceProvider),
+	fx.Provide(TaskQueueManagerProvider),
```

Comment:
add TODO here: it should be provided via persistence factory module.

Reply:
Done (also added internal task)